doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -23,6 +23,56 @@ reference these objects is not allowed.
+|---------------------------|------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `execution_id`            | [`uuid`]                     | An ID that is unique for each execute statement.                                                                                                                                                                                                                              |
+| `sample_rate`             | [`double precision`]         | The actual rate at which the statement was sampled.                                                                                                                                                                                                                           |
+| `cluster_id`              | [`text`]                     | The ID of the cluster the statement execution was directed to.                                                                                                                                                                                                                |
+| `application_name`        | [`text`]                     | The value of the `application_name` session variable at execution time.                                                                                                                                                                                                       |
+| `cluster_name`            | [`text`]                     | The name of the cluster with ID `cluster_id` at execution time.                                                                                                                                                                                                               |
+| `transaction_isolation`   | [`text`]                     | The value of the `transaction_isolation` session variable at execution time.                                                                                                                                                                                                  |
+| `execution_timestamp`     | [`uint8`]                    | The logical timestamp at which education was scheduled.                                                                                                                                                                                                                       |
```

Comment:
```suggestion
| `execution_timestamp`     | [`uint8`]                    | The logical timestamp at which execution was scheduled.                                                                                                                                                                                                                       |
```

Reply:
Similarly, I wish `execution_time` was the second column in this table. I always scramble to find the column.

Reply:
I agree, will change in a follow-up